tests/test_emulators.py

```diff
@@ -1060,3 +1072,142 @@ def test_simple(self):
+            "raven-routing-sample/VIC_streaminputs.nc"
+        )
+        vic_temperatures_nc_path = get_local_testdata(
+            "raven-routing-sample/VIC_temperatures.nc"
+        )
+
+        #########
+        # Model #
+        #########
+
+        model = Routing()  # "/tmp/ravenpy_routing_emu_dev")
+
+        #######
+        # RVI #
+        #######
+
+        streaminputs = xr.open_dataset(vic_streaminputs_nc_path)
+
+        start_date = streaminputs.time[0].dt
+        end_date = streaminputs.time[-1].dt
+
+        # Round to hours
+        n_seconds = (
+            ((streaminputs.time[-1] - streaminputs.time[0]) / len(streaminputs.time))
+            .dt.round("h")
+            .dt.seconds.item()
+        )
+        # Warning: this is only good up to 24 hours (24:00:00)
+        time_step = time.strftime("%H:%M:%S", time.gmtime(n_seconds))
+
+        model.rvi.start_date = to_py_datetime(start_date)
+        model.rvi.end_date = to_py_datetime(end_date)
+        model.rvi.time_step = time_step
+
+        #######
+        # RVH #
+        #######
+
+        rvh_importer = RoutingProductShapefileImporter(routing_product_shp_path)
+        (
+            subbasins,
+            land_group,
+            lake_group,
+            reservoirs,
+            channels,
+            hrus,
+        ) = rvh_importer.extract()
+
+        model.rvh.subbasins = subbasins
+        model.rvh.land_subbasins = land_group
+        model.rvh.lake_subbasins = land_group
+        model.rvh.reservoirs = reservoirs
+        model.rvh.hrus = hrus
+
+        #######
+        # RVP #
+        #######
+
+        model.rvp.avg_annual_runoff = 594
+        model.rvp.channel_profile_list = channels
+
+        #######
+        # RVT #
+        #######
+
+        streaminputs_importer = RoutingProductGridWeightImporter(
+            vic_streaminputs_nc_path, routing_product_shp_path
+        )
+
+        temperatures_importer = RoutingProductGridWeightImporter(
+            vic_temperatures_nc_path, routing_product_shp_path
+        )
+
+        streaminputs_gf = GriddedForcingCommand(
+            name="StreamInputs",
+            forcing_type="PRECIP",
+            file_name_nc=vic_streaminputs_nc_path.name,
+            var_name_nc="Streaminputs",
+            dim_names_nc=("lon_dim", "lat_dim", "time"),
+            grid_weights=streaminputs_importer.extract(),
+        )
+
+        temperatures_gf = GriddedForcingCommand(
+            name="AverageTemp",
+            forcing_type="TEMP_AVE",
+            file_name_nc=vic_temperatures_nc_path.name,
+            var_name_nc="Avg_temp",
+            dim_names_nc=("lon_dim", "lat_dim", "time"),
+            grid_weights=temperatures_importer.extract(),
+        )
+
+        model.rvt.gridded_forcings = [streaminputs_gf, temperatures_gf]
+
+        #############
+        # Run model #
+        #############
+
+        model([vic_streaminputs_nc_path, vic_temperatures_nc_path])
```

Comment:
Can we check that the model output matches the tutorial output ?

Reply:
@julemai and @dustming Could you propose a good strategy for this?

Reply:
> @julemai and @dustming Could you propose a good strategy for this?

Compare simulated hydrograph in Hydrographs.csv?

Reply:
compare with what?

Reply:
now done!